### DISCUSSION TITLE
Is it possible to generate a xcode project that supports BOTH ios simulator and real iPhone

### DISCUSSION THREAD

**Discussion by cchuicchui** [axmolengine/axmol#885] [Q&A]
Is it possible to generate a xcode project that supports BOTH ios simulator and real iPhone?

Current available options are

     for ios armv7,arm64 combined:
     cmake -S . -B build -GXcode -DCMAKE_TOOLCHAIN_FILE=$AX_ROOT/cmake/ios.toolchain.cmake -DPLATFORM=OS
     for ios simulator x86_64:
     cmake -S . -B build -GXcode -DCMAKE_TOOLCHAIN_FILE=$AX_ROOT/cmake/ios.toolchain.cmake -DPLATFORM=SIMULATOR64

My normal development environment is ios simulator, but it is not possible for me to deploy the test code to my iPhone for testing.

It is also impossible to build the app and deploy to apple store, if ios simulator is selected.

But if xcode is used, ios simulator is the main method of doing development and debugging.

This is my main obstacle of moving from cocos2d v4 to axmol. Cocos2d is supporting both ios simulator and real iphone in the same xcode project from the beginning to v4.

How can I do both using the same xcode project in axmol?

**Comment by paulocoutinhox** [comment]
I put the solution here:
https://github.com/axmolengine/axmol/issues/860

Need implementation.

**Reply by aismann** [reply]
> I put the solution here: #860
> 
> Need implementation.


@cchuicchui 
I think this is the answer. Can you set it plz?

**Reply by cchuicchui** [reply]
Will test this out today and will set answer shortly. Sorry for the delay

**Comment by cchuicchui** [comment]
Where should I add "-DAX_WITH_ASTC=OFF -DFT_DISABLE_BZIP2=ON"?

**Reply by paulocoutinhox** [reply]
When you create the cmake project.

**Comment by cchuicchui** [comment]
Thanks it works!

Axmol team should really consider supporting OS64COMBINED since this is the most sensible setup in xcode.
